src/components/AddTokenButton/AddTokenButton.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState, useMemo } from 'react'
+import styled from 'styled-components'
+import { SWPR } from '@swapr/sdk'
+import { useActiveWeb3React } from '../../hooks'
+import { useTokenBalance } from '../../state/wallet/hooks'
+
+const MAINNET_ADDRESS = '0x6cacdb97e3fc8136805a9e7c342d866ab77d0957'
+const ARBITRUM_ADDRESS = '0xde903e2712288a1da82942dddf2c20529565ac30'
+
+const Button = styled.button`
```

Comment:
I'd stylize the base button instead of the standard html button here. It has pointer cursor and removed outline, hich keep the UI consistent.